### ISSUE TITLE
Run cmor-fixer from arbitrary directory

### ISSUE THREAD

**Issue by uwefladrich** [EC-Earth/cmor-fixer#38]
When the `cmor-fixer` is run from another directory than it's checked out code base, it fails:
```
> Projects/cmor-fixer/cmor-fixer.py -h

 Error:  The netcdf data file  nemo-vertices/nemo-vertices-ORCA1-t-grid.nc   does not exist.
```
This can happen either by giving the path (as the example above) other than the current directory, or by putting the `cmor-fixer` somewhere in the `PATH`.

It would be good to allow the `cmor-fixer` to run from any directory or to mention the restriction in the `README`. It would be good to know if the `nemo-vertices` directory is the only file the fixer expects to find in the current directory.

**Comment by treerink** [comment]
Related to #37.

**Comment by treerink** [comment]
Fixed by 4b722d0 as suggested by @pchengi in #41.
